sql/filings-by-zip-table-outside-nyc.sql

```diff
@@ -0,0 +1,26 @@
+with 
+filings_zips as (
+	select 
+		i.*, 
+		LEFT(postalcode,5) AS zipcode,
+		court as court_name
+	from oca_index i 
+	left join oca_addresses a on a.indexnumberid = i.indexnumberid
+	where i.fileddate >= '03-23-2020' and i.classification = any('{Holdover,Non-Payment}') 
```

Comment:
Oh one other quick thing, the `fileddate` field is in YYYY-MM-DD format. I think PostgreSQL is smart enough to interpret your where clause appropriately as a date but it may be good to write it in that format just for clarity's sake? Or is there a reason it's better to use that format over the one that the data is presented in?

Reply:
Noted! revising